test/blackbox-tests/test-cases/github2033/run.t

```diff
@@ -0,0 +1,5 @@
+. should be allowed in c names
+  $ dune build
+  Info: creating file dune-project with this contents:
+  | (lang dune 1.9)
```

Comment:
the project file is missing in the test :)

Reply:
Thanks, I added one.